### DISCUSSION TITLE
Styles unsyncing / repeating leading to overall style issues

### DISCUSSION THREAD

**Discussion by binajmen** [vercel/next.js#23852] [Help]
### What version of Next.js are you using?

10.0.7

### What version of Node.js are you using?

14.15.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

I'm struggling for more than 48h with this issue, so I decided to overpass the feeling this is a specific issue from my side and to leverage your experience and brains as mine is 🤯 

FYI, my setup is NextJS + TypeScript + Chakra UI.

I have some kind of "styling unsyncing" that appeared recently, and I'm unable to find the cause.

What I noticed though is the weird amount of styles piling up. My first question would be: **is this normal?** The 
subsidiary question would be: **what would be the cause?**

![Screenshot 2021-04-09 at 09 55 52](https://user-images.githubusercontent.com/15611419/114149149-e11e7080-991a-11eb-85f1-124cc072d70b.png)
![Screenshot 2021-04-09 at 09 56 18](https://user-images.githubusercontent.com/15611419/114149157-e380ca80-991a-11eb-9c8a-64af23ca7fc9.png)
![Screenshot 2021-04-09 at 09 56 35](https://user-images.githubusercontent.com/15611419/114149166-e5e32480-991a-11eb-8190-21f9169099f0.png)
![Screenshot 2021-04-09 at 09 56 49](https://user-images.githubusercontent.com/15611419/114149172-e8457e80-991a-11eb-89f9-04689d9865cc.png)

![ezgif-7-346d530153f5](https://user-images.githubusercontent.com/15611419/114160128-e1246d80-9926-11eb-87b2-dd2dda21e78e.gif)

I also noticed that when transitioning from one page to another (using `Link` from `next/link` as a wrapper around a `button`), scroll is disabled if for example the button is in a `Drawer` that set body overflow to hidden. It seems the style from the `Drawer` is not correctly removed when transitioning to the second page.

### Expected Behavior

Having the styles working correctly

### To Reproduce

Unfortunately, I can't share the repo as it is private and not mine. I tried to reproduce the issue in a sandbox without success..

I know animated gif is not the greatest to help debugging, but I'm willing to share any code snippet you would suspect being the problem and hope it will ring a bell on what you experienced yourself on a past project 🙏 🙏

**Comment by Samin100** [comment]
I'm having the same issue where Chakra's styles aren't properly loading. Could you share your package.json?

**Comment by binajmen** [comment]
Hi @Samin100,

And did you manage to solve it ?

```js
// package.json

{
  "name": "temp",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start -p $PORT"
  },
  "dependencies": {
    "@chakra-ui/react": "^1.3.3",
    "@emotion/react": "^11.1.5",
    "@emotion/styled": "^11.1.5",
    "@nandorojo/swr-firestore": "^0.16.0",
    "dayjs": "^1.10.4",
    "easy-peasy": "^5.0.3",
    "emotion-theming": "^11.0.0",
    "firebase": "^8.2.7",
    "firebase-admin": "^9.5.0",
    "formik": "^2.2.6",
    "framer-motion": "^3.3.0",
    "geofire-common": "^5.1.0",
    "immer": "^8.0.1",
    "nanoid": "^3.1.22",
    "next": "^10.0.7",
    "next-firebase-auth": "^0.13.0-alpha.0",
    "next-translate": "^1.0.2",
    "react": "17.0.1",
    "react-beautiful-dnd": "^13.0.0",
    "react-datepicker": "^3.6.0",
    "react-dom": "17.0.1",
    "react-google-places-autocomplete": "^3.3.0",
    "react-icons": "^4.2.0",
    "react-redux": "^7.2.2",
    "use-sound": "^2.0.1",
    "yup": "^0.32.9"
  },
  "devDependencies": {
    "@types/node": "^14.14.27",
    "@types/react": "^17.0.2",
    "@types/react-beautiful-dnd": "^13.0.0",
    "@types/react-datepicker": "^3.1.7",
    "file-loader": "^6.2.0",
    "typescript": "^4.1.5",
    "url-loader": "^4.1.1"
  }
}
```

**Comment by binajmen** [comment]
I have a better understanding of the issue. It seems the prefetching cause the prefetched code to be executed as well, breaking the design.

I've created https://github.com/vercel/next.js/issues/23879 as it seems to me it should not execute prefetch code.

**Comment by binajmen** [comment]
It was due to emotion/react being loaded several time (buggy third party component)
